hilti/runtime/src/tests/library-dummy.cc

```diff
@@ -10,7 +10,8 @@ extern "C" {
 int foo() { return RETURN_VALUE; }
 }
 
-const char HILTI_EXPORT HILTI_WEAK* __hlt_hlto_library_version = R"({
+// NOTE: This should be consistent with the identifier generated with `HILTI_INTERNAL_GLOBAL_ID`.
+const char HILTI_EXPORT HILTI_WEAK* hlt_internal_hlto_library_version = R"({
```

Comment:
The macro should work here: 
```suggestion
const char HILTI_EXPORT HILTI_WEAK* HILTI_INTERNAL_GLOBAL(hlto_library_version) = R"({
```

(Needs to include `util.h` then.)